11-BLE-peripheral/main.c

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2025 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ * @brief       WSN_BLE_peripheral_01
```

Comment:
```suggestion
 * @brief       BLE_peripheral
```

It's not really WSN-specific I'd say. Also, to keep consistency with the other folder names, I'd rename the folder to lowercase (`11-ble-peripheral`).